frb_codegen/src/commands.rs

```diff
@@ -52,6 +98,9 @@ pub(crate) fn bindgen_rust_to_dart(arg: BindgenRustToDartArg) -> anyhow::Result<
         arg.dart_class_name,
         arg.llvm_install_path,
         arg.llvm_compiler_opts,
+        dart_root
+            .as_deref()
+            .unwrap_or(env!("CARGO_MANIFEST_DIR").into()),
```

Comment:
nit: seems to have seen such "maybe_dart_root.unwrap_or(cargo manifest dir)" twice. Maybe we can parse dart root once (i.e. fill in default dir) and use it everywhere.

Reply:
that's exactly what I thought, but in `lib_flutter_rust_bridge_codegen::config::Opts`, `dart_root` is marked as `Option`al : should it be required instead ?

Reply:
Or, shall we 

```
impl Opts {
  fn dart_root_or_default(&self) {
    self.dartroot.unwrapor(cargomanifestdir)
  }
}

Reply:
actually the reason why I ask is that there's a conditional check at the bottom of `frb_codegen` method which depends on `dart_root` original value as provided by `Opt` and other informations only available later on in the scope : `freezed` and `build_runner` as can be seen [here](https://github.com/fzyzcjy/flutter_rust_bridge/blob/a385b70b4b87fd67221c1788d378172026da1f05/frb_codegen/src/lib.rs#L142-L156).

Reply:
another part of the code that bewilders me a bit is [here](https://github.com/fzyzcjy/flutter_rust_bridge/blob/797040d5faee71cd9dac24fd49deb7b199d3505f/frb_codegen/src/config.rs#L177-L186). At some points it mention that there could be multiple `dart_root`s.

Reply:
That link is about RawOpts, not Opts :)

Reply:
ok it should be done.